src/main/kotlin/gov/nist/secauto/oscal/tools/server/core/JsonItemWriter.kt

```
@@ -0,0 +1,223 @@
+    private fun String.escapeJson(): String = buildString {
+        this@escapeJson.forEach { char ->
+            when (char) {
+                '"' -> append("\\\"")
+                '\\' -> append("\\\\")
+                '\b' -> append("\\b")
+                '\u000C' -> append("\\f")
+                '\n' -> append("\\n")
+                '\r' -> append("\\r")
+                '\t' -> append("\\t")
+                else -> if (char < ' ') {
+                    append(String.format("\\u%04x", char.code))
```

detekt

[warning] String.format("\\u%04x", char.code) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)